config/initializers/revision.rb

```diff
@@ -2,6 +2,7 @@
 file = Rails.root.join('REVISION')
 
 Rails.application.config.samson.revision =
```

Comment:
that should be `.tag` and not `.revision`

Reply:
also rename the file

Reply:
maybe just leave this alone :(

... I think we might need both since the rollbar/airbrake config expects a commit-ish and the fuzzy-tag might not work for them